ui/src/components/facets/HistogramFacet.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from "react";
+import VegaLite from "react-vega-lite";
+import { Handler } from "vega-tooltip";
+
+const styles = {};
+
+const baseSpec = {
+  $schema: "https://vega.github.io/schema/vega-lite/v3.json",
+  mark: "bar",
+  width: 400,
+  height: 300,
+  encoding: {
+    color: {
+      field: "dimmed",
+      type: "nominal",
+      scale: {
+        range: ["#4c78a8", "#aaafb7"]
+      },
+      legend: null,
+    },
+    tooltip: [
+      {
+        title: "Facet Value",
+        field: "facet_value",
+        type: "nominal",
+      },
+      {
+        title: "Count",
+        field: "count",
+        type: "quantitative",
+      }
+    ],
+    x: {},
+    y: {}
+  }
+}
+
+function isRange(name) {
```

Comment:
We may want to just add an `es_field_type` field to the `Facet` API object instead of doing this regex matching.

Reply:
I agree, can you do that in this PR?